source/settings/enable-dev-tools.txt

```diff
@@ -0,0 +1,111 @@
+.. _compass-enable-dev-tools:
+
+======================
+Toggle Chrome DevTools
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can toggle the ``enableDevTools`` option to control user access to Chrome 
+DevTools on |compass|. When you enable Chrome Devtools, you can view additional 
+diagnostic or debugging information from the |compass-short| interface. 
+
+About This Task
+---------------
+
+By default, |compass-short| enables :guilabel:`Chrome DevTools`.
```

Comment:
By default the "Enable DevTools" value is `false`, so the feature is disabled.

Reply:
Noted and updated.